### PR TITLE
Add account in initial json state

### PR DESCRIPTION
**Pull request by hansl**
And fix an issue with ledger.send for accounts iwth feature
0 (ledger).

### CODE REVIEW

**Comment by hansl** [comment]
@fmorency I'm going to at least add a test for the new feature before committing, but you can review the draft PR as is (I tested manually).

File: src/many-ledger/src/module.rs [hunk 1]
```diff
@@ -236,10 +235,22 @@ impl ledger::LedgerCommandsModuleBackend for LedgerModuleImpl {
...
         if from != sender {
-            return Err(error::unauthorized());
+            if let Some(account) = self.storage.get_account(from) {
+                if account
+                    .features
+                    .has_id(account::features::ledger::AccountLedger::ID)
+                {
+                    account.needs_role(
+                        sender,
+                        [account::Role::Owner, account::Role::CanLedgerTransact],
```
Comment (fmorency) [inline src/many-ledger/src/module.rs:246]: nit: We should document somewhere that the account only requires ONE of the listed roles to be granted access.

File: src/many-ledger/src/json.rs [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+#[derive(serde::Deserialize, Clone, Debug, Default)]
+pub struct FeatureJson {
+    pub id: u32,
+    pub arg: Option<serde_json::value::Value>,
+}
+
+impl FeatureJson {
+    pub fn try_into_ledger(&self) -> Option<Feature> {
+        if self.id == 0 {
```
Comment (fmorency) [inline src/many-ledger/src/json.rs:25]: `account::features::ledger::AccountLedger::ID`

File: src/many-ledger/src/json.rs [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                            .collect::<Result<BTreeSet<account::Role>, _>>()
+                            .expect("Invalid role.")
+                    })
+                })
+                .collect(),
+            features: self
+                .features
+                .iter()
+                .map(|f| {
+                    f.try_into_ledger()
```
Comment (fmorency) [inline src/many-ledger/src/json.rs:99]: We'll want to refactor this in the future

**Review by fmorency — APPROVED** [review]
Small comments to address. Else LGTM! I'm approving assuming comments are fixed.

File: src/many-ledger/src/module.rs (hunk 1 above)
Comment (hansl) [reply src/many-ledger/src/module.rs:246]: Yeah we should refactor the error to be more explicit.

**Comment by hansl** [comment]
@fmorency I refactored the feature parsing to be simpler.

**Review by fmorency — APPROVED** [review]
I like it!

**Pull request merged**